### PR TITLE
Encrypt file extension

### PR DESCRIPTION
**Pull request by michaelavila**
This should be self explanatory

- [x] Encrypt file extension
- [x] Change key length from 128 to 256

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
 SECRET_KEY_BASE=123
+BB_SECRET='this is a test secret'
```
Comment (brianriley) [inline .env]: Oh, I don't think this file should be checked in. The `.env.test` file can be, but the `.env` file is for our own personal use.

Comment (michaelavila) [inline .env]: Agreed, but it was already in there when I started this work, so I had left it. I removed it now though.

Comment (brianriley) [inline .env]: My bad. Didn't realize I had checked it in. Thanks for removing it!

**Comment by brianriley** [comment]
:100:

**Pull request merged**